cli/script/command-executor.ts

```diff
@@ -659,17 +785,36 @@ function formatDate(unixOffset: number): string {
     }
 }
 
-function printAppList(format: string, apps: App[], deploymentLists: string[][]): void {
+function printAppList(format: string, apps: App[], deploymentLists: string[][], owners: string[]): void {
     if (format === "json") {
         var dataSource: any[] = apps.map((app: App, index: number) => {
-            return { "name": app.name, "deployments": deploymentLists[index] };
+            return { "name": app.name, "deployments": deploymentLists[index], "owner": owners[index] };
         });
+
         printJson(dataSource);
     } else if (format === "table") {
-        var headers = ["Name", "Deployments"];
+        var headers = ["Name", "Deployments", "Owner"];
         printTable(headers, (dataSource: any[]): void => {
             apps.forEach((app: App, index: number): void => {
-                var row = [app.name, wordwrap(50)(deploymentLists[index].join(", "))];
+                var row = [app.name, wordwrap(50)(deploymentLists[index].join(", ")), owners[index]];
+                dataSource.push(row);
+            });
+        });
+    }
+}
+
+function printCollaboratorsList(format: string, collaborators: Collaborator[]): void {
+    if (format === "json") {
+        var dataSource: any[] = collaborators.map((collaborator: Collaborator) => {
+            return { "email": collaborator.email };
+        });
+
+        printJson(dataSource);
+    } else if (format === "table") {
+        var headers = ["Collaborators"];
```

Comment:
Should this column header be "E-mail address" or something similar? I think it's clear that the list represents the collaborators, since the dev would have had to enter that command to display it, so it might make sense to name the column closer to what the contents represents.